packages/components/src/components/table-stateless/component.tsx

```diff
@@ -136,6 +139,35 @@ export class KolTableStateless implements TableStatelessAPI {
 		validateTableSelection(this, value);
 	}
 
+	@Listen('keydown')
+	public handleKeyDown(event: KeyboardEvent) {
+		if (event.key === 'ArrowUp' || event.key === 'ArrowDown') {
+			const focusedElement = this.tableDivElement?.querySelector(':focus') as HTMLInputElement;
+			let index = this.checkboxRefs.indexOf(focusedElement);
+
+			if (focusedElement && this.checkboxRefs.includes(focusedElement)) {
```

Comment:
(Vorschlag) Könnte vereinfacht werden zu
```suggestion
			if (index > -1) {
```